matplotcheck/tests/test_points.py

```diff
@@ -0,0 +1,161 @@
+"""Tests for the vector module"""
+import numpy as np
+import matplotlib
+import pytest
+import matplotlib.pyplot as plt
```

Comment:
@nkorinek please move the matplotlib import under the abve matplotlib import to follow PEP8

Reply:
i just fixed it. super minor!